modules/web/src/app/shared/components/labels/template.html

```diff
@@ -16,15 +16,24 @@
 <div fxLayout="row"
      fxLayoutAlign=" center"
      fxLayoutGap="8px">
-  <mat-chip-list *ngIf="labelKeys.length > 0">
-    <div *ngFor="let labelKey of labelKeys; let i = index">
+
+  <mat-chip-list [ngClass]="{'hide-extra-labels': showHiddenLabels && oneLineLimit}"
```

Comment:
```suggestion
  <mat-chip-list [ngClass]="{'hide-extra-labels': oneLineLimit && !showHiddenLabels}"
```
`showHiddenLabels` condition is flipped for better readability.